upgrade/upgrade-4.1.3.php

```diff
@@ -26,5 +26,6 @@
  */
 function upgrade_module_4_1_3($object)
 {
-    return $object->unregisterHook('displayHome');
+    return $object->unregisterHook('displayHome') 
```

Comment:
You can remove $object->unregisterHook('displayHome')

Reply:
@okom3pom It is a part of @Progi1984's PR 116 https://github.com/PrestaShop/ps_googleanalytics/pull/116/files#diff-6b48dcebdefb30591797f7fcb3b79fd86e2a6769c105d610b9fd32e787ce971e

Reply:
You are right sorry